### PR TITLE
docs: logger doc refactor

### PR DESCRIPTION
**Pull request by Chever-John**
### Pre-submission checklist:

* [ ] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: docs/en/latest/plugins/http-logger.md [hunk 1]
```diff
@@ -54,6 +54,8 @@ This will provide the ability to send Log data requests as JSON objects to Monit
 
 The plugin supports the use of batch processors to aggregate and process entries(logs/data) in a batch. This avoids frequent data submissions by the plugin, which by default the batch processor submits data every `5` seconds or when the data in the queue reaches `1000`. For information or custom batch processor parameter settings, see [Batch-Processor](../batch-processor.md#configuration) configuration section.
 
+The plugin supports the use of batch processors to aggregate and process entries(logs/data) in a batch. This avoids frequent data submissions by the plugin, which by default the batch processor submits data every `5` seconds or when the data in the queue reaches `1000`. For information or custom batch processor parameter settings, see [Batch-Processor](../batch-processor.md#configuration) configuration section.
```
Comment (spacewander) [inline docs/en/latest/plugins/http-logger.md:57]: Duplicate?

**Review by spacewander — COMMENTED** [review]
Some files are not updated. Please use `rg batch_max_size   docs | rg optional` to find them.

**Review by spacewander — DISMISSED** [review]

File: docs/zh/latest/plugins/clickhouse-logger.md [hunk 2]
```diff
@@ -45,11 +45,10 @@ title: clickhouse-logger
 | password         | string  | 必须   |               |         | clickhouse的密码 。  |
 | timeout          | integer | 可选   | 3             | [1,...] | 发送请求后保持连接活动的时间。                   |
 | name             | string  | 可选   | "clickhouse logger" |         | 标识 logger 的唯一标识符。                     |
-| batch_max_size   | integer | 可选   | 100           | [1,...] | 设置每批发送日志的最大条数，当日志条数达到设置的最大值时，会自动推送全部日志到 `clickhouse` 。 |
-| max_retry_count  | integer | 可选   | 0             | [0,...] | 从处理管道中移除之前的最大重试次数。               |
-| retry_delay      | integer | 可选   | 1             | [0,...] | 如果执行失败，则应延迟执行流程的秒数。             |
 | ssl_verify       | boolean | 可选   | true          | [true,false] | 验证证书。             |
 
+The plugin supports the use of batch processors to aggregate and process entries(logs/data) in a batch. This avoids frequent data submissions by the plugin, which by default the batch processor submits data every `5` seconds or when the data in the queue reaches `1000`. For information or custom batch processor parameter settings, see [Batch-Processor](../batch-processor.md#configuration) configuration section.
```
Comment (juzhiyuan) [inline docs/zh/latest/plugins/clickhouse-logger.md:50]: Hi @Chever-John, please use Chinese here.

**Review by juzhiyuan — COMMENTED** [review]
Also, please update this PR's checklist :)

File: docs/zh/latest/plugins/clickhouse-logger.md (hunk 2 above)
Comment (Chever-John) [reply docs/zh/latest/plugins/clickhouse-logger.md:50]: Oh no.

**Review by LiteSun — DISMISSED** [review]

File: docs/zh/latest/plugins/clickhouse-logger.md (hunk 2 above)
Comment (Chever-John) [reply docs/zh/latest/plugins/clickhouse-logger.md]: I have fixed the problem and checked carefully to make sure there is no such fundamental problem
Please review again! @spacewander @juzhiyuan Thanks.

**Review by spacewander — APPROVED** [review]

**Review by tzssangglass — APPROVED** [review]

**Pull request merged**